cmd/dymint/commands/show_p2p_info.go

```diff
@@ -0,0 +1,119 @@
+package commands
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	rpctypes "github.com/tendermint/tendermint/rpc/jsonrpc/types"
+)
+
+var ip, port string // used for flags
+
+type peerInfo struct {
+	peerId             string
+	multiAddress       string
+	connectionDuration time.Duration
+}
+
+// ShowNodeIDCmd dumps node's ID to the standard output.
+var ShowP2PInfoCmd = &cobra.Command{
```

Comment:
if all it does is calling `rpc.NetInfo` than the implementation is redundant

it should be implemented on the rollapp/RDK similar to 

```
func getNodeStatus(clientCtx client.Context) (*coretypes.ResultStatus, error) {
	node, err := clientCtx.GetNode()
	if err != nil {
		return &coretypes.ResultStatus{}, err
	}

	return node.Status(context.Background())
}
```

but just calling `node.NetInfo`

Reply:
got it. implemented in the rdk in https://github.com/dymensionxyz/dymension-rdk/pull/420